tests/lax_numpy_test.py

```diff
@@ -1152,6 +1152,23 @@ def testPolySub(self, a_shape, b_shape, dtype):
     self._CompileAndCheck(jnp_fun, args_maker, check_dtypes=True)
 
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": "a_shape={} , order={}".format(
```

Comment:
Change this to `"testcase_name": "_shape={}_order={}".format(`

Testcase names should not contain spaces.